spec/untangled/client/impl/om_plumbing_spec.cljs

```diff
@@ -331,4 +331,6 @@
         (impl/mark-missing {:b [{:x {:data 1}} {:x {:data 2}}]} [{:b [:x]}]) =fn=> (has-leaves [[:b 0 :x] [:b 1 :x]])
         (impl/mark-missing {:b []} [:a {:b [:x]}]) =fn=> (has-leaves [[:b]])
         "unions are followed"
-        (impl/mark-missing {:a [{:x {:data 1}} {:y {:data 2}}]} [{:a {:b [:x] :c [:y]}}]) =fn=> (has-leaves [[:a 0 :x] [:a 1 :y]])))))
+        (impl/mark-missing {:a [{:x {:data 1}} {:y {:data 2}}]} [{:a {:b [:x] :c [:y]}}]) =fn=> (has-leaves [[:a 0 :x] [:a 1 :y]])
+        "unions don't break when there is no result"
```

Comment:
thanks for the recommendation, spec title updated. The message came because the motivation for this was because some code was breaking, but I agree with you that the "is" is better than the "isn't".